client/extensions/wp-job-manager/state/setup/reducer.js

```diff
@@ -33,7 +50,20 @@ export const nextStep = createReducer( {}, {
 	[ WP_JOB_MANAGER_WIZARD_NEXT_STEP ]: ( state, { siteId } ) => ( { ...state, [ siteId ]: true } ),
 } );
 
+/**
+ * Tracks the setup status for a particular site.
+ *
+ * @param  {Object} state Current setup status
+ * @param  {Object} action Action object
+ * @return {Object} Updated setup status
+ */
+export const status = createReducer( {}, {
+	[ WP_JOB_MANAGER_UPDATE_SETUP_STATUS ]: ( state, { siteId, data } ) => ( { ...state, [ siteId ]: data } ),
```

Comment:
So I've heard overusing `data` as the name for 'all things data' is less then ideal, perhaps we should consider changing the variable/property to `status` so it's more descriptive ?

Reply:
The function is already named `status` so I'll have a conflict. Let me poke around and see how others name things here.

Reply:
Right. Naming always ends up being the most tricky part of what we do 😄   
How about going with `showWizard` for now, as it's just a boolean ? Or `completed` ? Though that would require an additional negation I guess.

If it takes too long just leave it 🙂

Reply:
I ended up doing this - https://github.com/Automattic/wp-calypso/pull/18314/commits/2e77bb3017429ab204011212b2ced06167273086. Could have also just used `status` and not de-structured it in the params. 🤷‍♀️